test/builder-haddock/default.nix

```diff
@@ -1,14 +1,10 @@
-{ mkPkgSet, stdenv }:
+{ mkCabalProjectPkgSet, stdenv }:
 
 with stdenv.lib;
 
 let
-  pkgSet = mkPkgSet {
-    # generated with:
-    #   cabal new-build
-    #   plan-to-nix dist-newstyle/cache/plan.json > plan.nix
-    #   cabal-to-nix test-haddock.cabal > test-haddock.nix
-    pkg-def = import ./plan.nix;
+  pkgSet = mkCabalProjectPkgSet {
+    pkg-def = import ./pkgs.nix;
```

Comment:
I think this needs to be `plan-pkgs`, similar to how `mkStackPkgSet` has `stack-pkgs`

Reply:
Ah yes, thanks, I forgot to update them all ...